plugins/org.eclipse.fordiac.ide.fb.interpreter/src/org/eclipse/fordiac/ide/fb/interpreter/inputgenerator/BaseRandom.java

```diff
@@ -36,7 +36,7 @@
 import org.eclipse.fordiac.ide.model.data.WcharType;
 import org.eclipse.fordiac.ide.model.data.WordType;
 
-public abstract class BaseRandom {
+public abstract class BaseRandom implements BaseRandomNextValue {
```

Comment:
why is this better then the previous version

Reply:
Honestly: I did not want to just add 17 `@SuppressWarning('static-method')` annotations without looking for a alternative solution

Reply:
I can live with that.